pages/index.html

```diff
@@ -4,9 +4,40 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Abby Choi</title>
+  <link href="/styles/style.css" rel="stylesheet">
+  <link href="/styles/index.css" rel="stylesheet">
+  
 </head>
+
 <body>
-  
+<div class="container">
+<header>
+  <h1>Abby Choi</h1>
+  <nav>
+    <ul>
+      <li><a id="index" href="/pages/index.html">Home</a></li>
+      <li><a href="/pages/about.html">About Me</a></li>
+      <li><a href="/pages/portfolio.html">Portfolio</a></li>
```

Comment:
These links to pages and styles work perfectly if the site is run through a server, but not by opening the static HTML without the server. This is fine, but something to be aware of in case you might experience this in the future.